test-programs/registry-test/tests/tests.rs

```diff
@@ -58,38 +58,38 @@ async fn test_initialize_protocol_config() {
     let mut rpc = ProgramTestRpcConnection { context: rpc };
 
     let payer = rpc.get_payer().insecure_clone();
-    let program_account_keypair = Keypair::from_bytes(&REGISTRY_ID_TEST_KEYPAIR).unwrap();
+    let program_account_keypair = Keypair::from_bytes(&OLD_REGISTRY_ID_TEST_KEYPAIR).unwrap();
     let protocol_config = ProtocolConfig::default();
     let (protocol_config_pda, bump) = get_protocol_config_pda_address();
     let ix_data = light_registry::instruction::InitializeProtocolConfig {
         protocol_config,
         bump,
     };
 
-    // init with invalid authority
-    {
-        let accounts = light_registry::accounts::InitializeProtocolConfig {
-            protocol_config_pda,
-            authority: payer.pubkey(),
-            fee_payer: payer.pubkey(),
-            system_program: solana_sdk::system_program::id(),
-            self_program: light_registry::ID,
-        };
-        let ix = Instruction {
-            program_id: light_registry::ID,
-            accounts: accounts.to_account_metas(Some(true)),
-            data: ix_data.data(),
-        };
-        let result = rpc
-            .create_and_send_transaction(&[ix], &payer.pubkey(), &[&payer])
-            .await;
-        assert_rpc_error(
-            result,
-            0,
-            anchor_lang::error::ErrorCode::ConstraintRaw as u32,
-        )
-        .unwrap();
-    }
+    // // init with invalid authority
```

Comment:
Same here

Reply:
Yes it's commented because I commented the onchain check which makes this test fail see notes in description.